src/components/UrlUtilsService.js

```diff
@@ -48,13 +48,44 @@ goog.provide('ga_urlutils_service');
                   /.*kmz$/.test(url));
         };
 
-        this.proxifyUrl = function(url) {
+        // Test using a head request if the remote resource enables CORS
+        this.isCorsEnabled = function(url) {
+          var deferred = $q.defer();
+          $http.head(url, { timeout: 1500 }).then(function(response) {
+            // If the head request responded successfully it means that CORS
+            // is enabled
```

Comment:
```
$ curl -I https://www.wandern.ch/ewalc/images/ewalk2/696/geodata/geodata_de.xml
HTTP/1.1 200 OK
Date: Fri, 10 Mar 2017 09:46:32 GMT
Server: Apache
Last-Modified: Mon, 30 Jan 2017 20:13:09 GMT
ETag: "4d8d-5475571c34058"
Accept-Ranges: bytes
Content-Length: 19853
Vary: Accept-Encoding
NZSpeedy: OFF
NZX: OK
Content-Type: application/xml
```

They don't support CORS. It's blocked by the browser as expected and we have to use ogcproxy.


https://www.gipfelbuch.ch/global/kml/gipfel/1198/show

Now we have an issue when requesting a resource that has been moved, like the one above (redirection over http).
So we still need to handle this case.

Again this needs to go through ogcproxy as expected. I don